pipeline/backend/docker/convert_test.go

```diff
@@ -0,0 +1,49 @@
+package docker
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestSplitVolumeParts(t *testing.T) {
+	testdata := []struct {
+		from    string
+		to      []string
+		success bool
+	}{
+		{
+			from:    `Z::Z::rw`,
+			to:      []string{`Z:`, `Z:`, `rw`},
+			success: true,
+		},
+		{
+			from:    `Z:\:Z:\:rw`,
+			to:      []string{`Z:\`, `Z:\`, `rw`},
+			success: true,
+		},
+		{
+			from:    `Z:\git\refs:Z:\git\refs:rw`,
+			to:      []string{`Z:\git\refs`, `Z:\git\refs`, `rw`},
+			success: true,
+		},
+		{
+			from:    `Z:\git\refs:Z:\git\refs`,
+			to:      []string{`Z:\git\refs`, `Z:\git\refs`},
```

Comment:
What about things like `Z:/git`? It seemed thats also an option on Windows.

Reply:
It is a test directory.

Reply:
It's OK "Z:/git". I added more test cases.